pae-examples.xhtml

```diff
@@ -137,13 +137,13 @@ version-footer: true
                 <div class="highlight"><pre><code id="code_pae4"></code></pre></div>
             </p>
 
-            <h4>Grace notes (accaciature)</h4>
+            <h4>Grace notes (accacciature)</h4>
```

Comment:
thanks for checking through! I didn't want to change to english on my own, but you're probably right.